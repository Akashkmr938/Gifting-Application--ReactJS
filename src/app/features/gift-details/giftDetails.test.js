import React from 'react';
import renderer from 'react-test-renderer';
import GiftDetails from './giftDetails';

describe('The GiftDetails component', () => {
    const props ={
        location: {
            search: ''
        }
    }
    it('renders correctly', () => {
        const tree = renderer
            .create(<GiftDetails {...props}/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
})