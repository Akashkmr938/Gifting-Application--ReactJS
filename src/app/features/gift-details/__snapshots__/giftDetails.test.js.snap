// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The GiftDetails component renders correctly 1`] = `null`;
